.travis.yml

```
@@ -11,6 +11,9 @@ matrix:
         - env: TOXENV=py38
           python: 3.8
           dist: xenial  # Required for Python >= 3.7 (travis-ci/travis-ci#9069)
+        - env: TOXENV=py39
+          python: 3.9
+          dist: focal  # Required for Python >= 3.7 (travis-ci/travis-ci#9069)
```

```suggestion
          dist: focal
```
That issue was closed when Xenial became the default distro